app/types.ts

```diff
@@ -140,3 +140,11 @@ export interface QuizSession {
   stream: string,
   id: string
 }
+
+export interface SessionSchedule {
```

Comment:
this really exactly matches with SessionOccurrence haha

Reply:
shouldn't a batch_id be here somewhere

Reply:
Added!